your_daily_dose_malware/backends/malshare.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import requests
+import click
+from dotenv import load_dotenv
+from rich.progress import Progress
+
+load_dotenv()
+
+
+def scrap_malshare():
+    with Progress() as progress:
+        api_key = os.getenv("MALSHARE_API_KEY")
+        print(api_key)
+        try:
+            response = requests.post(
+                "https://malshare.com/api.php",
```

Comment:
Can we collate website URLs etc. as constants in [Constants.py](https://github.com/Anti-Malware-Alliance/your-daily-dose-malware/blob/main/your_daily_dose_malware/app/utils/constants.py)?

Reply:
Constant files are more ued in other languages, in Python they are uncommon, they make the codeel harder to read. Im fine keeping it as is